docs/components/grapesjs.rst

```diff
@@ -0,0 +1,74 @@
+GrapesJS Builder
+################
+
+Currently, Mautic uses the GrapesJS Builder for Emails and Landing pages. 
+
+Setup 
+*****
+
+.. code-block:: BASH
+
+    npm install
+
+.. vale off
+
+Configure Babel, ESLint, Prettier
+=================================
+
+.. vale on
+
+Use the template files provided. For example .eslintrc.temp
```

Comment:
And do what with them? Can we be a bit more helpful here? Even if we point people out to documentation on external sites, it would be helpful to folks who might not know what to do with the template files.